web/admin/src/request/types.ts

```diff
@@ -1298,6 +1193,58 @@ export interface V1DeleteUserReq {
   user_id: string;
 }
 
+export interface V1FeishuGetDocReq {
```

Comment:
在types.ts文件中，部分接口定义可以进一步完善以提高类型安全性。

```suggestion
// 建议为必填字段添加明确的类型定义
export interface V1FeishuGetDocReq {
  // 明确指定doc_id为必填字段
  doc_id: string;
  file_type?: string;
  // 明确指定id为必填字段
  id: string;
  // 明确指定kb_id为必填字段
  kb_id: string;
  space_id?: string;
}